### PR TITLE
Feature/list of talks

### PR DESCRIPTION
**Pull request by MeenaFoerderer**

### CODE REVIEW

File: components/TalkCard/TalkCard.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+//import { data } from "../utils/data";
```
Comment (mpagels) [inline components/TalkCard/TalkCard.js:1]: A lonely comment here. ;-) Recommend deleting this.

File: components/TalkCard/TalkCard.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+export default function TalkCard() {
+  return (
+    <>
+      <h2>JURASSIC TALK 2022 Schedule</h2>
+      <StyledUl>
+        {data?.map((talk) => (
+          <StyledLi key={talk.id}>
+            <h4>{talk.title}</h4>
+            <h4>{talk.authors[0]}</h4>
```
Comment (mpagels) [inline components/TalkCard/TalkCard.js:13]: Please use descening headline tags. Read more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements?retiredLocale=de) about it.

File: components/TalkCard/TalkCard.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            <StyledDiv>
+              <p>{talk.session}</p>
+              <p>{talk.time}</p>
+              <p>{talk.room}</p>
+            </StyledDiv>
```
Comment (mpagels) [inline components/TalkCard/TalkCard.js:18]: I think `StyledDiv`it is not a good name. Maybe you find some more meaningful name for the styled component. Like `TalkInfoWrapper`

**Review by mpagels — APPROVED** [review]
Please, please provide a pull request description, where you tell the reviewer, what this pull request is about and put some additional infos in here. For example, reviewers might skip the review, because they see, that there are 585!!!!!! lines of addition in here. But if you tell them, "peepz, that are 550 Lines of fake data" there might be taking a look. ;-) 

Another thing I saw, that you're naming your styled components like `Styled...`. In my opinion that is not good. Try to find some meaningful names. For example `TalkList` and `TalkItems`. It is easier to understand what this list is about and easier to read and understand in the codebase.

That said: Awesome work here. A very good start. Keep up!!

File: components/TalkCard/TalkCard.js (hunk 3 above)
Comment (MeenaFoerderer) [reply components/TalkCard/TalkCard.js:18]: Thank you for the helpful review!

**Pull request merged**